### PR TITLE
#9 more generic regex

### PR DESCRIPTION
**Pull request by aanersc**

### CODE REVIEW

**Comment by aanersc** [comment]
@akorosov 
@aperrin66 
Please accept this PR first in order to be able to work with harvesting repo!

File: tests/normalizers/test_sentinel1_identifier.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def setUp(self):
         ]
         self.normalizer = normalizers.sentinel1_identifier.SentinelOneIdentifierMetadataNormalizer(
             DATASET_PARAMETER_NAMES)
-
+        #MATCHER=self.normalizer.MATCHER
```
Comment (aperrin66) [inline tests/normalizers/test_sentinel1_identifier.py]: Please remove this

File: tests/normalizers/test_sentinel1_identifier.py [hunk 2]
```diff
@@ -125,3 +125,15 @@ def test_return_none_for_incorrect_raw_attribute(self):
...
+        """ Shall return None based on non-recognizable raw attribute """
+
+        result_normalization = self.normalizer.match_identifier({'Identifier': 'S1A_EW_GRD__1SDH_20150702T172954_20150702T173054_006635_008DA5_55D1'})
+        self.assertEqual(result_normalization['resolution'], "_")
+
+        result_normalization = self.normalizer.match_identifier({'Identifier': 'S1A_EW_GRDM_1SDH_20150702T172954_20150702T173054_006635_008DA5_55D1'})
+        self.assertEqual(result_normalization['resolution'], "M")
+
+        result_normalization = self.normalizer.match_identifier({'Identifier': 'S1A_EW_GRDM_1SDH_20150702T172954_20150702T173054________008DA5_55D1'})
+        self.assertEqual(result_normalization['orbit'], "______")
```
Comment (aperrin66) [inline tests/normalizers/test_sentinel1_identifier.py:138]: Ideally, we would need to test each field with a real value and with underscores.

File: tests/normalizers/test_sentinel1_identifier.py (hunk 1 above)
Comment (aanersc) [reply tests/normalizers/test_sentinel1_identifier.py]: Yeah,this is my mistake to remove this line during developement.

**Comment by aanersc** [comment]
@aperrin66 This https://github.com/nansencenter/metanorm/pull/12#discussion_r453587752 is addressed in ba7989c.

**Pull request merged**